src/views/movieDetailsView/movieDetailsView.js

```diff
@@ -0,0 +1,211 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+import CustomButton from '../../components/common/customButton/customButton'
+import WikiBadge from '../../components/common/customBadges/customBadges'
+
+import { fetchMovieDetails } from '../../actions/movieDetails';
+import { goBack } from '../../actions/navigation';
+
+import { API_PARAMS } from '../../common/constants'
+
+import './movieDetailsView.scss';
+
+
+class MoviesDetailsView extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showPlot: false
```

Comment:
As it's Boolean - good practice is to have it as `isPlotVisible` or something similar starting with `is` or `should`.